### PR TITLE
Profile charts menu

### PR DESCRIPTION
**Pull request by airenzp**
## Description

Displayed Facility Radar and Radar buttons with menu upon click to select the radar wanted. Removes profile specific logic in the mass/charts. Removed profileHome and older logic. Added function onFilterChange that will be called to clear the local filters in the profile.
See corresponding PR in sjpp.

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/plots/profilePlot.js [hunk 1]
```diff
@@ -624,6 +623,10 @@ export async function loadFilterTerms(config, app, opts) {
 	config.filterTWs = twlst
```
Comment (siosonel) [inline client/plots/profilePlot.js:623]: Syntax improvement: in `twlst.push()` in line 609 or 608, no need to create an array and then spread it again, should just be 
```js
twlst.push(
	config.countryTW,
	config.regionTW,
	config.incomeTW,
	config.typeTW,
	config.teachingStatusTW,
	config.referralStatusTW,
	config.fundingSourceTW,
	config.hospitalVolumeTW,
	config.yearOfImplementationTW
)
```

Comment (airenzp) [reply client/plots/profilePlot.js:621]: thanks done

File: client/mass/store.js [hunk 2]
```diff
@@ -334,6 +334,8 @@ TdbStore.prototype.actions = {
 				parent.lst[i] = replacementFilter
 			}
 		}
+		//In the profile runproteinpaint this function is passed to clear the local filters when the global filter changes
+		if (this.app.opts.app?.onFilterChange) this.app.opts.app.onFilterChange(this.state.plots)
```
Comment (xzhou82) [inline client/mass/store.js:338]: is this constructor option? i would like this documented somewhere, if not in a script then in client/mass/README

File: client/plots/profilePlot.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ import { select } from 'd3-selection'
 import { getSampleFilter } from '../mass/groups.js'
 import { Menu } from '#dom/menu'
 import { icons as icon_functions } from '#dom/control.icons'
+import { config } from 'process'
```
Comment (xzhou82) [inline client/plots/profilePlot.js:9]: what's this for?

Comment (airenzp) [reply client/plots/profilePlot.js]: It got introduced by mistake by VS Code autocomplete. Removed now.

File: client/mass/store.js (hunk 2 above)
Comment (airenzp) [reply client/mass/store.js:338]: It is used in the profile index and I add some comments there.

Comment (airenzp) [reply client/mass/store.js:338]: There is no client/mass/README.md, should I add one?

Comment (airenzp) [reply client/mass/store.js:338]: I added some comments in mass/app. Please check.

**Review by siosonel — APPROVED** [review]

**Pull request merged**